README.md

```diff
@@ -30,6 +30,20 @@ $ jq '.[].Link' data/data.json | sort | uniq | wc
 # oneliner to find the number of duplicates
 $ echo $((`jq '.[].Link' data/data.json | sort | uniq -D | wc -l`-`jq '.[].Link' data/data.json | sort | uniq -d | wc -l`))
 ```
+#### Getting all paper links from new library site
+
+Run the following from the `data` folder:
+
+```sh
+python ../scripts/pdfFinder.py lib_data.json
+```
+(You need to install BS4 for that.)
+
+This will generate a new `lib_data.json` (overwriting the old one).
+Prettify the json and commit.
+Once `data/data.json` is updated with the links from crowd-sourced question papers,
+we can just copy-paste the categorized entries from `lib_data.json` to `data.json`. 
+However, the uncategorized links must be scrutinized and converted to proper format before adding.
```

Comment:
I made some modifications from the scripts, so that it infers the unavailable data for uncategorized links from the parent links. I will make the relevant changes in the Readme.

Reply:
@grapheo12 Please write the commands that can be used to update the file using the above script. You can also do that you take in the items of the json file, append your new items in the same format, check for duplicates using list to set conversion and then save the json data. All in a single script. That will make it really easy to use later as well.